File: Data Structures & Algorithms/majority-element/submission-1.js
class Solution {
    /**
     * @param {number[]} nums
     * @return {number}
     */
    majorityElement(nums) {
        let ans = {};
        let largest = 0;
        for(let i = 0; i < nums.length; i++){
            if(nums[i] in ans){
                ans[nums[i]] += 1;
            } else if (!(nums[i] in ans)){
                ans[nums[i]] = 1;
            }       
        }
        let most_key = 0;
        let most = 0;
        for(let key in ans){
            let value = ans[key];
            if(value > most){
                most = value;
                most_key = key;
            }
        }
        return most_key;
    }
}

File: Data Structures & Algorithms/isomorphic-strings/submission-1.py
class Solution:
    def isIsomorphic(self, s: str, t: str) -> bool:
        ans = {}
        for i in range(len(s)):
            if s[i] in ans:
                if ans[s[i]] != t[i]:
                    return False
            else:
                for key in ans:
                    if ans[key] == t[i]:
                        return False
                ans[s[i]] = t[i]
        return True